pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -485,6 +496,20 @@ async def _notify_microshift_alerts(self, version_release: str):
 @click_coroutine
 async def build_microshift(runtime: Runtime, data_path: str, group: str, assembly: str, payloads: Tuple[str, ...],
                            no_rebase: bool, force: bool):
-    pipeline = BuildMicroShiftPipeline(runtime=runtime, group=group, assembly=assembly, payloads=payloads,
-                                       no_rebase=no_rebase, force=force, data_path=data_path)
-    await pipeline.run()
+    # slack client is dry-run aware and will not send messages if dry-run is enabled
+    slack_client = runtime.new_slack_client()
+    slack_client.bind_channel(group)
+    try:
+        pipeline = BuildMicroShiftPipeline(runtime=runtime, group=group, assembly=assembly, payloads=payloads,
+                                           no_rebase=no_rebase, force=force, data_path=data_path, slack_client=slack_client)
+        await pipeline.run()
+    except Exception as err:
+        slack_message = f"build-microshift pipeline encountered error: {err}"
+        reaction = None
+        error_message = slack_message + f"\n {traceback.format_exc()}"
+        runtime.logger.error(error_message)
+        if assembly not in ["stream", "test", "microshift"]:
```

Comment:
the assembly here is release_name

Reply:
that's right, we don't have assembly type available here, since that requires fetching release config